.changelog/1202.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/cloudflare_account_member: fix the issue - provider not able to detect changes made via UI for account member role Id's
+```
```

Comment:
```suggestion
```release-note:enhancement
resource/cloudflare_account_member: handle role changes made in the dashboard
```
```